src/ex4/server/services/item_manager.js

```diff
@@ -1 +1,89 @@
-// The ItemManager should go here. Remember that you have to export it.
\ No newline at end of file
+const fs = require('fs').promises;
```

Comment:
the common convention of files in javascript projects is kebab-case and not snake_case
```
item_manager.js -> item-manager.js
```